src/lib/support/BUILD.gn

```diff
@@ -73,6 +73,42 @@ source_set("attributes") {
   ]
 }
 
+source_set("logging_header") {
+  sources = [ "logging/CHIPLogging.h" ]
+
+  public_deps = [
+    ":attributes",
+    ":logging_constants",
+    "${chip_root}/src/lib/core:chip_config_header",
+  ]
+
+  if (chip_pw_tokenizer_logging) {
+    public_deps += [ "${dir_pw_tokenizer}" ]
+  }
+}
+
+source_set("codeutils") {
```

Comment:
utils is a very poor name. Can we do better? 

This library is already support, adding core/utils/helpers adds more words without meaning to it.
Maybe `verifymacros` for codeutils, `errorstring` or similar for errorstr.

This does seem to split this into a lot of source sets, but that seems to be a sideffect that we started with something named `support` that is a grab-all of everything.

Reply:
Sure, I can split into "errorstring" and "verifymacros".  The latter is a bit non-intuitive if you look for "what do I need to depend on to get CodeUtils.h", but in practice I don't expect anything out side this .gn file to look at these fine-grained dependencies.